tests/org.eclipse.swt.tests.win32/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,8 @@ Bundle-Version: 3.108.100.qualifier
 Bundle-Vendor: Eclipse.org
 Bundle-Localization: plugin
 Require-Bundle: org.junit;bundle-version="4.12.0",
- org.eclipse.swt
+ org.eclipse.swt,
+ org.eclipse.swt.tests
```

Comment:
The problem with such a dependency is that it only makes sense in the context of a parallel Maven build. It does something else than what these dependency specifications are supposed to be used for, so no one except the ones involved in this PR will know that (and probably even they will forget), so there will be people trying to understand the necessity for this dependency and eventually even remove it.

I would at least suggest to add a comment in the Manifest that explains the necessity for the dependency.

Reply:
> I would at least suggest to add a comment in the Manifest that explains the necessity for the dependency.

Manifest can't have comments. Another alternative would be additional.bundles in the `build.properties` or a dependency in the `pom.xml` or a `p2.inf`

Reply:
> Manifest can't have comments

Just out of interest: did that change at some point in time? Because in a former project, several years ago, I had comments in a Manifest (but I do not remember the syntax).

Reply:
> > Manifest can't have comments
> 
> Just out of interest: did that change at some point in time? Because in a former project, several years ago, I had comments in a Manifest (but I do not remember the syntax).

It didn't change. But an OSGi framework is supposed to ignore Headers it doesn't understand. So you can have a `Comment: Some remark` header as long as all characters are permitted.

In general I agree with your concern, that's why I added an explanation in the commit message (that is then hopefully found).
But moving it to the build.properties is probably the best solution here.
Will prepare a follow-up tonight.

Reply:
A header is not a comment... manifests are meant to be processed by tools not humans that why [the specification](https://docs.oracle.com/en/java/javase/20/docs/specs/jar/jar.html#manifest-specification) is rather "odd" and its wide use makes its near to impossible to change (e.g. the stupid no more than 72 bytes (**not characters**) rule).

Reply:
Thanks for clarification. Now I remember that we used a `Comment: ...` header, knowing that it was actually a misuse.

Reply:
Created https://github.com/eclipse-platform/eclipse.platform.swt/pull/1017.